docker-compose/cse-import-d2cc/config/cse-d2cc-gridcapa-runner-application.yml

```diff
@@ -9,6 +9,8 @@ spring:
     template:
       reply-timeout: 7200000
   cloud:
+    function:
```

Comment:
Why did you add this here, and not in any other process-runner-configuration ?

Reply:
You're right, that's a mistake that happened while i was experimenting changes required. It should have been deleted afterwards